port/board/afc-v3/payload.c

```diff
@@ -477,7 +477,7 @@ uint8_t payload_hpm_activate_firmware( void )
     return IPMI_CC_OK;
 }
 
-void clock_configuration()
+mmc_err clock_configuration()
```

Comment:
Signature doesn't match implementation, it should be `mmc_err clock_configuration(void)`.